components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -762,13 +786,19 @@ public AccessTokenDO retrieveAccessToken(String accessTokenIdentifier, boolean i
                 OAuth2Util.checkUserNameAssertionEnabled()) {
                 userStoreDomain = OAuth2Util.getUserStoreDomainFromAccessToken(accessTokenIdentifier);
             }
-            String sql = SQLQueries.RETRIEVE_ACTIVE_ACCESS_TOKEN;
+
+            String sql;
+
             if (includeExpired) {
                 sql = SQLQueries.RETRIEVE_ACTIVE_EXPIRED_ACCESS_TOKEN;
+            } else {
+                sql = SQLQueries.RETRIEVE_ACTIVE_ACCESS_TOKEN;
             }
```

Comment:
Why is this change needed ? Earlier it was initialized at declaration and changed under if condition. Now it is initialized under if or else. Is this change needed in some way. If not remove it.